witchcraft/internal/wdebug/handlers.go

```diff
@@ -26,20 +26,27 @@ import (
 	"github.com/palantir/pkg/metrics"
 	werror "github.com/palantir/witchcraft-go-error"
 	wparams "github.com/palantir/witchcraft-go-params"
+	"github.com/palantir/witchcraft-go-server/v2/witchcraft/wdebug"
 )
 
+// DiagnosticHandler is deprecated in favor of the exported/non-external wdebug.DiagnosticHandler.
```

Comment:
```suggestion
// DiagnosticHandler is deprecated in favor of the exported wdebug.DiagnosticHandler.
```

What does "/non-external" mean here? It looks like that interface is exported/external.

Reply:
thanks for catching, fixed